src/components/Appbar/Appbar.js

```diff
@@ -29,89 +26,26 @@ const ApplicationBar = (props) => {
-            {simplified ? (
-              <>
-                <Anchor color={"inherit"} underline={"none"} style={{cursor: 'pointer'}}>
-                <Link onClick={backToHome} activeClass='active' spy={true} smooth={true} duration={500} offset={20}
-                  style={{
-                    display: "flex",
-                    alignItems: "center",
-                    gap: ".5rem",
-                  }}
-                
-                >
-                  <img
-                    src={logo}
-                    alt="logo"
-                    style={{ width: "50px", height: "50px" }}
-                  />
-                  <Typography variant="h6" component="div">
-                    {title}
-                  </Typography>
-                  </Link>
-                </Anchor>
-              </>
-            ) : (
-              <>
-                <img
-                  src={logo}
-                  alt="logo"
-                  style={{ width: "50px", height: "50px" }}
-                />
-                <Typography variant="h6" component="div">
-                  {title}
-                </Typography>
-              </>
-            )}
-          </Box>
-          {!simplified ? (
-            <>
-              <Box style={{ display: "flex", flexGrow: 1, gap: "2rem", alignItems: 'center'}}>
-                {navbarData.map((item, index) => {
-                  return (
-                    <>
-                      {item.dropdown === false ? (
-                        <>
-                        <Anchor color={'inherit'} underline={'none'} style={{cursor: 'pointer'}} key={index}>
-                          <Link to={item.to} activeClass='active' spy={true} smooth={true} duration={500} offset={-70}>
-                            <Typography variant="h6" component="div" className='link'>
-                              {item.link}
-                            </Typography>
-                            </Link>
-                          </Anchor>
-                        </>
-                      ) : (
-                        <>
-                          <Box style={{display:'flex', justifyContent: 'center', alignItems: 'center', cursor: 'pointer'}}>
-                          <AppDropdown dropdownTitle={item.link} optionsArray={shopButton}/>
-                          </Box>
-                        </>
-                      )}
-                    </>
-                  );
-                })}
-              </Box>
-            </>
-          ) : (
-            <></>
-          )}
-          <AppModal buttonName={"SIGN IN"} buttonColor={'button-black'} title={"ACCOUNT LOGIN"} description={<SystemLogin/>} 
-          buttonStyle={{fontSize: '15px', padding: '10px 25px', borderRadius: '30px', fontWeight: 600}} logo/>
-        </Toolbar>
-      </AppBar>
+      <header className={`${bg ? 'bg-primary h-20' : 'h-24'} flex items-center fixed top-0 w-full text-black dark:text-white z-10 transition-all duration-300`}>
```

Comment:
As per the changes , did we remove our re-usable application bar here and replace it with header instead ?

Reply:
Yes, appbar from material ui was removed and changed it into header and className for the implementation of tailwind. Or do you want to revert appbar and modify it's classsname?

Reply:
If we can much better.

Reply:
Noted with thanks